### PR TITLE
use requirePage for /_document and /_app

### PR DESCRIPTION
**Pull request by timneutkens**
Uses the pagesManifest instead of creating the full path

### CODE REVIEW

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall decrease ✓</summary>

|  | vercel/next.js canary  | timneutkens/next.js add/remove-buildid-to-hash | Change |
| - | - | - | - |
| buildDuration | 12.1s | 12.5s | ⚠️ +429ms |
| nodeModulesSize | 65.2 MB | 65.2 MB | -1.28 kB |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall decrease ⚠️</summary>

|  | vercel/next.js canary  | timneutkens/next.js add/remove-buildid-to-hash | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.136 | 2.17 | ⚠️ +0.03 |
| / avg req/sec | 1170.59 | 1152.19 | -18.4 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.34 | 1.514 | ⚠️ +0.17 |
| /error-in-render avg req/sec | 1866.13 | 1650.84 | -215.29 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js add/remove-buildid-to-hash | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.51 kB | 6.51 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..0538.js gzip | 10.5 kB | 10.5 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 56.9 kB | 56.9 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js add/remove-buildid-to-hash | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.6 kB | 5.6 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..dule.js gzip | 6.93 kB | 6.93 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 52.4 kB | 52.4 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js add/remove-buildid-to-hash | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.3 kB | 26.3 kB | ✓ |
| Overall change | 26.3 kB | 26.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js add/remove-buildid-to-hash | Change |
| - | - | - | - |
| routerDirect.js gzip | 279 B | 279 B | ✓ |
| withRouter.js gzip | 278 B | 278 B | ✓ |
| _error.js gzip | 3.37 kB | 3.37 kB | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 2.05 kB | 2.05 kB | ✓ |
| hooks.js gzip | 881 B | 881 B | ✓ |
| _app.js gzip | 1.26 kB | 1.26 kB | ✓ |
| Overall change | 8.34 kB | 8.34 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js add/remove-buildid-to-hash | Change |
| - | - | - | - |
| index.module.js gzip | 223 B | 223 B | ✓ |
| routerDirect..dule.js gzip | 281 B | 281 B | ✓ |
| withRouter.m..dule.js gzip | 278 B | 278 B | ✓ |
| hooks.module.js gzip | 383 B | 383 B | ✓ |
| _error.module.js gzip | 2.21 kB | 2.21 kB | ✓ |
| link.module.js gzip | 1.52 kB | 1.52 kB | ✓ |
| _app.module.js gzip | 604 B | 604 B | ✓ |
| Overall change | 5.49 kB | 5.49 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js add/remove-buildid-to-hash | Change |
| - | - | - | - |
| _buildManifest.js gzip | 270 B | 270 B | ✓ |
| _buildManife..dule.js gzip | 274 B | 274 B | ✓ |
| Overall change | 544 B | 544 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js add/remove-buildid-to-hash | Change |
| - | - | - | - |
| index.html gzip | 956 B | 956 B | ✓ |
| link.html gzip | 963 B | 963 B | ✓ |
| withRouter.html gzip | 949 B | 949 B | ✓ |
| Overall change | 2.87 kB | 2.87 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall decrease ✓</summary>

|  | vercel/next.js canary  | timneutkens/next.js add/remove-buildid-to-hash | Change |
| - | - | - | - |
| buildDuration | 13.3s | 13.2s | -107ms |
| nodeModulesSize | 65.2 MB | 65.2 MB | -1.28 kB |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js add/remove-buildid-to-hash | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.51 kB | 6.51 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..0538.js gzip | 10.5 kB | 10.5 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 56.9 kB | 56.9 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js add/remove-buildid-to-hash | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.6 kB | 5.6 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..dule.js gzip | 6.93 kB | 6.93 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 52.4 kB | 52.4 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js add/remove-buildid-to-hash | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.3 kB | 26.3 kB | ✓ |
| Overall change | 26.3 kB | 26.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js add/remove-buildid-to-hash | Change |
| - | - | - | - |
| routerDirect.js gzip | 279 B | 279 B | ✓ |
| withRouter.js gzip | 278 B | 278 B | ✓ |
| _error.js gzip | 3.37 kB | 3.37 kB | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 2.05 kB | 2.05 kB | ✓ |
| hooks.js gzip | 881 B | 881 B | ✓ |
| _app.js gzip | 1.26 kB | 1.26 kB | ✓ |
| Overall change | 8.34 kB | 8.34 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js add/remove-buildid-to-hash | Change |
| - | - | - | - |
| index.module.js gzip | 223 B | 223 B | ✓ |
| routerDirect..dule.js gzip | 281 B | 281 B | ✓ |
| withRouter.m..dule.js gzip | 278 B | 278 B | ✓ |
| hooks.module.js gzip | 383 B | 383 B | ✓ |
| _error.module.js gzip | 2.21 kB | 2.21 kB | ✓ |
| link.module.js gzip | 1.52 kB | 1.52 kB | ✓ |
| _app.module.js gzip | 604 B | 604 B | ✓ |
| Overall change | 5.49 kB | 5.49 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js add/remove-buildid-to-hash | Change |
| - | - | - | - |
| _buildManifest.js gzip | 270 B | 270 B | ✓ |
| _buildManife..dule.js gzip | 274 B | 274 B | ✓ |
| Overall change | 544 B | 544 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong> Overall decrease ✓</summary>

|  | vercel/next.js canary  | timneutkens/next.js add/remove-buildid-to-hash | Change |
| - | - | - | - |
| _error.js | 875 kB | 875 kB | -236 B |
| 404.html | 4.17 kB | 4.17 kB | ✓ |
| hooks.html | 3.79 kB | 3.79 kB | ✓ |
| index.js | 875 kB | 875 kB | -236 B |
| link.js | 913 kB | 913 kB | -236 B |
| routerDirect.js | 906 kB | 905 kB | -236 B |
| withRouter.js | 906 kB | 905 kB | -236 B |
| Overall change | 4.48 MB | 4.48 MB | -1.18 kB |

</details>

</details>

File: packages/next/build/index.ts [hunk 1]
```diff
@@ -502,47 +496,30 @@ export default async function build(dir: string, conf = null): Promise<void> {
...
+
+      const serverBundle = getPagePath(page, distDir, isLikeServerless)
 
       let isSsg = false
       let isStatic = false
       let isHybridAmp = false
       let ssgPageRoutes: string[] | null = null
       let hasSsgFallback: boolean = false
 
-      pagesManifest[page] = bundleRelative.replace(/\\/g, '/')
```
Comment (timneutkens) [inline packages/next/build/index.ts:521]: @Timer @ijjk  I'm not sure why this line was here. It seems unneeded as it overrides the existing pagesManifest path which was generated during the build

**Comment by ijjk** [comment]
## Failing test suites 

**test/integration/bigint/test/index.test.js** 
- bigint API route support > serverless mode > should return 200
- bigint API route support > serverless mode > should return the BigInt result text

<details>
<summary>Expand output</summary>

  ● bigint API route support › serverless mode › should return 200

    command failed with code 1

      132 |         code !== 0
      133 |       ) {
    > 134 |         return reject(new Error(`command failed with code ${code}`))
          |                       ^
      135 |       }
      136 | 
      137 |       resolve({

      at ChildProcess.<anonymous> (lib/next-test-utils.js:134:23)

  ● bigint API route support › serverless mode › should return the BigInt result text

    command failed with code 1

      132 |         code !== 0
      133 |       ) {
    > 134 |         return reject(new Error(`command failed with code ${code}`))
          |                       ^
      135 |       }
      136 | 
      137 |       resolve({

      at ChildProcess.<anonymous> (lib/next-test-utils.js:134:23)

</details>

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall decrease ✓</summary>

|  | vercel/next.js canary  | timneutkens/next.js add/remove-buildid-to-hash | Change |
| - | - | - | - |
| buildDuration | 10.6s | 10.7s | ⚠️ +167ms |
| nodeModulesSize | 65.2 MB | 65.2 MB | -2.56 kB |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall decrease ⚠️</summary>

|  | vercel/next.js canary  | timneutkens/next.js add/remove-buildid-to-hash | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 1.838 | 1.83 | -0.01 |
| / avg req/sec | 1360.38 | 1366.14 | ⚠️ +5.76 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.051 | 1.1 | ⚠️ +0.05 |
| /error-in-render avg req/sec | 2379.25 | 2272.75 | -106.5 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js add/remove-buildid-to-hash | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.51 kB | 6.51 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..0538.js gzip | 10.5 kB | 10.5 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 56.9 kB | 56.9 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js add/remove-buildid-to-hash | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.6 kB | 5.6 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..dule.js gzip | 6.93 kB | 6.93 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 52.4 kB | 52.4 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js add/remove-buildid-to-hash | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.3 kB | 26.3 kB | ✓ |
| Overall change | 26.3 kB | 26.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js add/remove-buildid-to-hash | Change |
| - | - | - | - |
| routerDirect.js gzip | 279 B | 279 B | ✓ |
| withRouter.js gzip | 278 B | 278 B | ✓ |
| _error.js gzip | 3.37 kB | 3.37 kB | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 2.05 kB | 2.05 kB | ✓ |
| hooks.js gzip | 881 B | 881 B | ✓ |
| _app.js gzip | 1.26 kB | 1.26 kB | ✓ |
| Overall change | 8.34 kB | 8.34 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js add/remove-buildid-to-hash | Change |
| - | - | - | - |
| index.module.js gzip | 223 B | 223 B | ✓ |
| routerDirect..dule.js gzip | 281 B | 281 B | ✓ |
| withRouter.m..dule.js gzip | 278 B | 278 B | ✓ |
| hooks.module.js gzip | 383 B | 383 B | ✓ |
| _error.module.js gzip | 2.21 kB | 2.21 kB | ✓ |
| link.module.js gzip | 1.52 kB | 1.52 kB | ✓ |
| _app.module.js gzip | 604 B | 604 B | ✓ |
| Overall change | 5.49 kB | 5.49 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js add/remove-buildid-to-hash | Change |
| - | - | - | - |
| _buildManifest.js gzip | 270 B | 270 B | ✓ |
| _buildManife..dule.js gzip | 274 B | 274 B | ✓ |
| Overall change | 544 B | 544 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js add/remove-buildid-to-hash | Change |
| - | - | - | - |
| index.html gzip | 956 B | 956 B | ✓ |
| link.html gzip | 963 B | 963 B | ✓ |
| withRouter.html gzip | 949 B | 949 B | ✓ |
| Overall change | 2.87 kB | 2.87 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall decrease ✓</summary>

|  | vercel/next.js canary  | timneutkens/next.js add/remove-buildid-to-hash | Change |
| - | - | - | - |
| buildDuration | 11.8s | 11.3s | -419ms |
| nodeModulesSize | 65.2 MB | 65.2 MB | -2.56 kB |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js add/remove-buildid-to-hash | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.51 kB | 6.51 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..0538.js gzip | 10.5 kB | 10.5 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 56.9 kB | 56.9 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js add/remove-buildid-to-hash | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.6 kB | 5.6 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..dule.js gzip | 6.93 kB | 6.93 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 52.4 kB | 52.4 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js add/remove-buildid-to-hash | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.3 kB | 26.3 kB | ✓ |
| Overall change | 26.3 kB | 26.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js add/remove-buildid-to-hash | Change |
| - | - | - | - |
| routerDirect.js gzip | 279 B | 279 B | ✓ |
| withRouter.js gzip | 278 B | 278 B | ✓ |
| _error.js gzip | 3.37 kB | 3.37 kB | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 2.05 kB | 2.05 kB | ✓ |
| hooks.js gzip | 881 B | 881 B | ✓ |
| _app.js gzip | 1.26 kB | 1.26 kB | ✓ |
| Overall change | 8.34 kB | 8.34 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js add/remove-buildid-to-hash | Change |
| - | - | - | - |
| index.module.js gzip | 223 B | 223 B | ✓ |
| routerDirect..dule.js gzip | 281 B | 281 B | ✓ |
| withRouter.m..dule.js gzip | 278 B | 278 B | ✓ |
| hooks.module.js gzip | 383 B | 383 B | ✓ |
| _error.module.js gzip | 2.21 kB | 2.21 kB | ✓ |
| link.module.js gzip | 1.52 kB | 1.52 kB | ✓ |
| _app.module.js gzip | 604 B | 604 B | ✓ |
| Overall change | 5.49 kB | 5.49 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js add/remove-buildid-to-hash | Change |
| - | - | - | - |
| _buildManifest.js gzip | 270 B | 270 B | ✓ |
| _buildManife..dule.js gzip | 274 B | 274 B | ✓ |
| Overall change | 544 B | 544 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong> Overall decrease ✓</summary>

|  | vercel/next.js canary  | timneutkens/next.js add/remove-buildid-to-hash | Change |
| - | - | - | - |
| _error.js | 875 kB | 875 kB | -236 B |
| 404.html | 4.17 kB | 4.17 kB | ✓ |
| hooks.html | 3.79 kB | 3.79 kB | ✓ |
| index.js | 875 kB | 875 kB | -236 B |
| link.js | 913 kB | 913 kB | -236 B |
| routerDirect.js | 906 kB | 905 kB | -236 B |
| withRouter.js | 906 kB | 905 kB | -236 B |
| Overall change | 4.48 MB | 4.48 MB | -1.18 kB |

</details>

</details>

**Review by Timer — APPROVED** [review]

**Review by ijjk — APPROVED** [review]

File: packages/next/build/index.ts (hunk 1 above)
Comment (ijjk) [reply packages/next/build/index.ts:521]: Yeah it should be un-needed since the correct path should already be in the `pages-manifest` 👍

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall decrease ✓</summary>

|  | vercel/next.js canary  | timneutkens/next.js add/remove-buildid-to-hash | Change |
| - | - | - | - |
| buildDuration | 13s | 13.2s | ⚠️ +175ms |
| nodeModulesSize | 65.2 MB | 65.2 MB | -2.56 kB |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall decrease ⚠️</summary>

|  | vercel/next.js canary  | timneutkens/next.js add/remove-buildid-to-hash | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.265 | 2.242 | -0.02 |
| / avg req/sec | 1103.89 | 1115 | ⚠️ +11.11 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.428 | 1.496 | ⚠️ +0.07 |
| /error-in-render avg req/sec | 1750.52 | 1671.14 | -79.38 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js add/remove-buildid-to-hash | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.51 kB | 6.51 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..0538.js gzip | 10.5 kB | 10.5 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 56.9 kB | 56.9 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js add/remove-buildid-to-hash | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.6 kB | 5.6 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..dule.js gzip | 6.93 kB | 6.93 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 52.4 kB | 52.4 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js add/remove-buildid-to-hash | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.3 kB | 26.3 kB | ✓ |
| Overall change | 26.3 kB | 26.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js add/remove-buildid-to-hash | Change |
| - | - | - | - |
| routerDirect.js gzip | 279 B | 279 B | ✓ |
| withRouter.js gzip | 278 B | 278 B | ✓ |
| _error.js gzip | 3.37 kB | 3.37 kB | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 2.05 kB | 2.05 kB | ✓ |
| hooks.js gzip | 881 B | 881 B | ✓ |
| _app.js gzip | 1.26 kB | 1.26 kB | ✓ |
| Overall change | 8.34 kB | 8.34 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js add/remove-buildid-to-hash | Change |
| - | - | - | - |
| index.module.js gzip | 223 B | 223 B | ✓ |
| routerDirect..dule.js gzip | 281 B | 281 B | ✓ |
| withRouter.m..dule.js gzip | 278 B | 278 B | ✓ |
| hooks.module.js gzip | 383 B | 383 B | ✓ |
| _error.module.js gzip | 2.21 kB | 2.21 kB | ✓ |
| link.module.js gzip | 1.52 kB | 1.52 kB | ✓ |
| _app.module.js gzip | 604 B | 604 B | ✓ |
| Overall change | 5.49 kB | 5.49 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js add/remove-buildid-to-hash | Change |
| - | - | - | - |
| _buildManifest.js gzip | 270 B | 270 B | ✓ |
| _buildManife..dule.js gzip | 274 B | 274 B | ✓ |
| Overall change | 544 B | 544 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js add/remove-buildid-to-hash | Change |
| - | - | - | - |
| index.html gzip | 956 B | 956 B | ✓ |
| link.html gzip | 963 B | 963 B | ✓ |
| withRouter.html gzip | 949 B | 949 B | ✓ |
| Overall change | 2.87 kB | 2.87 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall decrease ✓</summary>

|  | vercel/next.js canary  | timneutkens/next.js add/remove-buildid-to-hash | Change |
| - | - | - | - |
| buildDuration | 14.2s | 14.5s | ⚠️ +293ms |
| nodeModulesSize | 65.2 MB | 65.2 MB | -2.56 kB |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js add/remove-buildid-to-hash | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.51 kB | 6.51 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..0538.js gzip | 10.5 kB | 10.5 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 56.9 kB | 56.9 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js add/remove-buildid-to-hash | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.6 kB | 5.6 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..dule.js gzip | 6.93 kB | 6.93 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 52.4 kB | 52.4 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js add/remove-buildid-to-hash | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.3 kB | 26.3 kB | ✓ |
| Overall change | 26.3 kB | 26.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js add/remove-buildid-to-hash | Change |
| - | - | - | - |
| routerDirect.js gzip | 279 B | 279 B | ✓ |
| withRouter.js gzip | 278 B | 278 B | ✓ |
| _error.js gzip | 3.37 kB | 3.37 kB | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 2.05 kB | 2.05 kB | ✓ |
| hooks.js gzip | 881 B | 881 B | ✓ |
| _app.js gzip | 1.26 kB | 1.26 kB | ✓ |
| Overall change | 8.34 kB | 8.34 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js add/remove-buildid-to-hash | Change |
| - | - | - | - |
| index.module.js gzip | 223 B | 223 B | ✓ |
| routerDirect..dule.js gzip | 281 B | 281 B | ✓ |
| withRouter.m..dule.js gzip | 278 B | 278 B | ✓ |
| hooks.module.js gzip | 383 B | 383 B | ✓ |
| _error.module.js gzip | 2.21 kB | 2.21 kB | ✓ |
| link.module.js gzip | 1.52 kB | 1.52 kB | ✓ |
| _app.module.js gzip | 604 B | 604 B | ✓ |
| Overall change | 5.49 kB | 5.49 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js add/remove-buildid-to-hash | Change |
| - | - | - | - |
| _buildManifest.js gzip | 270 B | 270 B | ✓ |
| _buildManife..dule.js gzip | 274 B | 274 B | ✓ |
| Overall change | 544 B | 544 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong> Overall decrease ✓</summary>

|  | vercel/next.js canary  | timneutkens/next.js add/remove-buildid-to-hash | Change |
| - | - | - | - |
| _error.js | 875 kB | 875 kB | -236 B |
| 404.html | 4.17 kB | 4.17 kB | ✓ |
| hooks.html | 3.79 kB | 3.79 kB | ✓ |
| index.js | 875 kB | 875 kB | -236 B |
| link.js | 913 kB | 913 kB | -236 B |
| routerDirect.js | 906 kB | 905 kB | -236 B |
| withRouter.js | 906 kB | 905 kB | -236 B |
| Overall change | 4.48 MB | 4.48 MB | -1.18 kB |

</details>

</details>

**Pull request merged**